net/freeradius/src/opnsense/service/templates/OPNsense/Freeradius/mods-enabled-ldap

```diff
@@ -146,7 +148,16 @@ ldap {
 
         }
         tls {
+		{%     if helpers.exists('OPNsense.freeradius.ldap.ldapstarttls') and OPNsense.freeradius.ldap.ldapstarttls == '1' %}
+                start_tls = yes
+        {%     else %}
+                start_tls = no
+        {%     endif %}
                 require_cert    = 'allow'
+        {%     if helpers.exists('OPNsense.freeradius.ldap.ldapcert') and OPNsense.freeradius.ldap.ldapcert != '' %}
+                certificate_file = ${certdir}/cert_ldap.pem
```

Comment:
Both pointing to the same file is ok?

Reply:
yes, because it's to be inserted only it's present a certificate to use, so if exist and if it's not null.